studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -49,8 +49,8 @@ const NavigationBar = ({}) => {
       ].join(' ')}
     >
       <ul className="flex flex-col space-y-2">
-        {!navLayoutV2 && (
-          <Link href="/projects">
+        {(!navLayoutV2 || IS_PLATFORM) && (
```

Comment:
Is this check correct? Getting the following on platform:

<img width="221" alt="Screenshot 2023-07-31 at 3 16 00 pm" src="https://github.com/supabase/supabase/assets/10985857/971d7331-d7e7-45bc-b5f5-e4475b2dd343">